### ISSUE TITLE
Flask Sqlachemy model Json fields missing

### ISSUE THREAD

**Issue by APinchofDill** [AtomHash/evernode#39]
If model is not loaded using a query filter statement the class is an instance and will only show attributes if explicitly targeted. 

Json and json_model will require new code to handle these circumstances. 
```
from sqlalchemy import inspect

if hasattr(Model, '_sa_instance_state'):
mapper = inspect(Model)
for column in mapper.attrs:
    print(column.key)
    print(column.value)
```

**Comment by APinchofDill** [comment]
For example:
```
model = Model()
model.field_name= "Name"
model.save()
Json_Response(200, None, model) 
```
This will respond with {} for model.

**Comment by APinchofDill** [comment]
Fixed in commit d2dfb799b18f89563f0227bb02236f673676228a
